x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/utils.ts

```diff
@@ -263,3 +266,9 @@ export const getCspmCloudShellDefaultValue = (packageInfo: PackageInfo): string
 
   return cloudShellUrl;
 };
+
+export const isBelowMinVersion = (version: string, minVersion: string) => {
```

Comment:
nit: `isBelowVersion`, whether it is min or max version has nothing to do with this utility. right?